docs/upgrade/UPGRADE-unreleased.md

```diff
@@ -16,7 +16,47 @@ There you can find links to upgrade notes for other versions too.
 - check whether you extended class or method `ImageFacade::copyImages` or used it in your project and make sure it works like you intended ([#851](https://github.com/shopsys/shopsys/pull/851))
     - *(low priority)* remove `/var/www/html/var/cache` folder from your `@main_filesystem` filesystem storage if exists, set as local filesystem storage in path `%kernel.project_dir%` by default
 - use `\Shopsys\FrameworkBundle\Component\Money\Money` class for representing monetary values in the whole application ([#821](https://github.com/shopsys/shopsys/pull/821))
-    - detailed upgrade instruction will be added shortly, see the PR for more information
+    - if you have created your custom entities that store monetary value, change your `decimal` ORM type to `money` type
+    ```diff
+    - * @var string
+    - * @ORM\Column(type="decimal", precision=20, scale=6)
+    + * @var \Shopsys\FrameworkBundle\Component\Money\Money
+    + * @ORM\Column(type="money", precision=20, scale=6)
+    ```
+        - update your custom data objects annotation type to `\Shopsys\FrameworkBundle\Component\Money\Money`
+    - if you are creating value for these custom entities or data objects directly, you have to use `Money::create` method that accepts string or integer or `Money::createFromFloat` method that accepts float and has second mandatory parameter scale
+    - if you need to compute mathematical operations use `Money::add`, `Money::substract`, `Money::multiply` or `Money::divide` method
+    - if you need to compare two `Money` use `compare`, `equals`, `isGreaterThan`, `isGreaterThanOrEqualTo`, `isLessThan` or `isLessThanOrEqualTo` method
+    - if you need to use `Money` in javascript, it is automatically serialized to JSON object with `amount` parameter
+    - `MoneyType` form type has been updated, it now works with `Money` class and uses `NumericToMoneyTransformer`, if you are using it you have to:
+        - use one of these two new constraints:
+            - `Shopsys\FrameworkBundle\Form\Constraints\MoneyRange` for setting acceptable range (only one side of interval is required)
+            - `Shopsys\FrameworkBundle\Form\Constraints\NotNegativeMoneyAmount` for setting that entered amount of money has to be zero or higher
+            - if non of these constraints is sufficient for your use case, create your own ([see commit with ne MoneyRange constraint](https://github.com/shopsys/shopsys/pull/821/commits/390b21416e54ca8bf34a7f7deec4e5c3eed5cd51))
```

Comment:
```suggestion
            - if non of these constraints is sufficient for your use case, create your own ([see commit with MoneyRange constraint](https://github.com/shopsys/shopsys/pull/821/commits/390b21416e54ca8bf34a7f7deec4e5c3eed5cd51))
```

Reply:
Please don't make links to a specific commit. The commit hash could change once the commit is rebased (luckily not this case, but in general it is a true)

Reply:
Why not link the specific commit in this case then? It won't ever change as it is merged to master already and it is a perfect working example of how such a constraint is implemented (with JS validation as well).

Reply:
I expresses myself wrongly... It is a bad idea to link a specific commit because it caused us already couple of problems because we rebase a lot.

In this particular case it is ok